### ISSUE TITLE
Error deleting snapshot resources manually

### ISSUE THREAD

**Issue by cmanzur** [elastic/terraform-provider-elasticstack#37]
This is a **bug** report:

1- Create this resource and `terraform apply`
```
resource "elasticstack_elasticsearch_snapshot_repository" "s3" {
  name = "s3"

  s3 {
    bucket = "backups"
    compress  = true
  }
}
```

2- Manually delete the repository from KIBANA UI.

3- Run `terraform apply` again

```
Error: Unable to get the information about snapshot repository: s3

Failed with:
{"error":{"root_cause":[{"type":"repository_missing_exception","reason":"[s3]
missing"}],"type":"repository_missing_exception","reason":"[s3]
missing"},"status":404}
```

Same applies for: **elasticstack_elasticsearch_snapshot_lifecycle**

**Notes:**
To use s3, you must install the plugin: `bin/elasticsearch-plugin install --batch repository-s3`

ES version: **docker.elastic.co/elasticsearch/elasticsearch:7.16.3**

**Comment by olksdr** [comment]
Hey @cmanzur, thanks for reporting this issue!  I'll be looking into this in the beginning of the next week.

**Comment by olksdr** [comment]
New version of the provider was published, see: [v0.2.0](https://github.com/elastic/terraform-provider-elasticstack/releases/tag/v0.2.0)
